src/chainstate/stacks/db/blocks.rs

```diff
@@ -173,17 +175,29 @@ impl FromRow<StagingBlock> for StagingBlock {
         let sortition_burn_i64 : i64 = row.get("sortition_burn");
         let block_data : Vec<u8> = vec![];
 
+        if commit_burn_i64 < 0 {
```

Comment:
Do you think it's worthwhile for us to implement `FromSql<u64>` and `ToSql<u64>` ? It'd reduce some of the repetitive error handling for getting u64s into and out of the database.

Reply:
Yes, thanks.

Reply:
Will do this in a separate PR: https://github.com/blockstack/blockstack-core/issues/1260